dags/public-health/Code75.py

```diff
@@ -0,0 +1,76 @@
+from datetime import datetime, timedelta
+
+import arcgis
+
+from airflow import DAG
+from airflow.hooks.base_hook import BaseHook
+from airflow.hooks.postgres_hook import PostgresHook
+from airflow.operators.python_operator import PythonOperator
+
+default_args = {
+    "owner": "airflow",
+    "depends_on_past": False,
+    "start_date": datetime(2020, 1, 30),
+    "email": [
+        "ian.rose@lacity.org",
+        "hunter.owens@lacity.org",
+        "brendan.bailey@lacity.org",
+    ],
+    "email_on_failure": True,
+    "email_on_retry": False,
+    "retries": 1,
+    "retry_delay": timedelta(days=1),
+}
+
+dag = DAG("code-75", default_args=default_args, schedule_interval="@daily")
+
+
+def update_code_75(**kwargs):
+    # Connecting to ArcGIS
+    arcconnection = BaseHook.get_connection("arcgis")
+    gis = arcgis.GIS(
+        "http://lahub.maps.arcgis.com/", arcconnection.login, arcconnection.password
```

Comment:
I had a bit of trouble testing this because the login/password we currently have in the "arcgis" connection doesn't seem to have access to this item.

Reply:
Sorry about that! Let me reach out to Oscar to add the dummy user to the group

Reply:
Ok - the user should now have access to the layers

Reply:
Great! Can you green CI with https://github.com/CityOfLosAngeles/aqueduct/pull/134#issuecomment-582026097 ?

Reply:
Sorry but what does "green CI" mean?

Reply:
Sorry, I just meant fixing the formatting check so that continuous-integration passes. Really just splitting that long URL with an implicit string concatenation.

Reply:
No problem! Done!